### PR TITLE
import and adapt the `run_pacta` script

### LINKED ISSUE #19: feat: add run_pacta.R
the part that runs standard pacta in bulk for all loan books/groups

AB#11599

### PR DESCRIPTION
**Pull request by cjyetman**
- closes #19

### CODE REVIEW

File: run_pacta.R [hunk 1]
```diff
@@ -0,0 +1,382 @@
+# load packages----
+# library(dotenv)
```
Comment (jacobvjk) [inline run_pacta.R]: nit: remove

File: run_pacta.R [hunk 2]
```diff
@@ -0,0 +1,382 @@
...
+
+
+# source helpers----
+source("expected_columns.R")
+source("plots.R")
+
+# load config----
+config <- config::get()
+
+input_path_scenario <- config$directories$dir_scenario
```
Comment (jacobvjk) [inline run_pacta.R:16]: nit: I have renamed some of the objects created from the confg in the other scripts, basically stripping the prefix `input_`. s anything that is a full path to a file is `path_* <- x/y.ext` and anything that is a directory is `dir_* <- x/y`. Not technically necessary to merge this, but maybe a good idea to use one style across the scripts.

File: run_pacta.R [hunk 3]
```diff
@@ -0,0 +1,382 @@
...
+input_dir_abcd <- config$directories$dir_abcd
+input_path_matched <- config$directories$dir_matched
+
+input_path_scenario_tms <- file.path(input_path_scenario, config$file_names$filename_scenario_tms)
+input_path_scenario_sda <- file.path(input_path_scenario, config$file_names$filename_scenario_sda)
+
+input_path_abcd <- file.path(input_dir_abcd, "abcd_final.csv")
+
+output_path <- config$directories$dir_output
```
Comment (jacobvjk) [inline run_pacta.R:25]: nit: see above, this would be `dir_output` now

**Review by jacobvjk — APPROVED** [review]
a few nits related to how I ended up renaming paths and directories, but feel free to ignore if all of the config yet up gets wrapped into a dedicated function soon anyway

File: run_pacta.R (hunk 1 above)
Comment (cjyetman) [reply run_pacta.R]: 💯 
```suggestion
```

File: run_pacta.R (hunk 2 above)
Comment (cjyetman) [reply run_pacta.R:16]: 💯 agree, but I think I'll wait to do that in a separate repo-wide review/alignment

Comment (cjyetman) [reply run_pacta.R:16]: - see #54

File: run_pacta.R (hunk 3 above)
Comment (cjyetman) [reply run_pacta.R:25]: 👍🏻 
- see #54

**Pull request merged**